Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN dotnet test
 FROM build AS publish
 RUN dotnet publish "DopplerCustomDomain/DopplerCustomDomain.csproj" -c Release -o /app/publish
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim AS final
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-buster-slim AS final
```

Comment:
I known that this is not the line for my suggestion. Do you need to expose 443 port ?
![image](https://user-images.githubusercontent.com/75738115/107365571-bc914e00-6abb-11eb-8bbf-39e460fa05a1.png)

Reply:
Fixed in #66